lib/src/countly_flutter.dart

```diff
@@ -193,7 +193,7 @@ class Countly {
 
           Countly.instance._remoteConfigInternal.notifyVariantCallbacks(requestResult, error, id);
         } catch (e) {
-          Countly.log(e.toString(), logLevel: LogLevel.ERROR);
+          Countly.log('[FMethodCallH] $e', logLevel: LogLevel.ERROR);
```

Comment:
can you check if this can be just 'log' and if so remove other Countly.logs plz

Reply:
yeah, done.